storage/zswap.txt

```diff
@@ -0,0 +1,92 @@
+ZSWAP [0]
+________________________________________________________________________________
+
+Zswap is a feature built-in to the Linux kernel [1] which allows users to 
+utilize compressed caches in RAM as swap pages. Instead of the kernel 
+immediately swapping pages in RAM and writing them to the hard drive, wasting 
+time on I/O operations, it compresses them into a pool in RAM. Only when the 
+available RAM is exhausted will the kernel then write-out the least recently 
+used page as an uncompressed file to swap on the drive.
+
+When swap will be used, zswap should provide performance improvements.
+
+
+Prerequisites
+________________________________________________________________________________
+
+Zswap works in conjuction with swap and is handled entirely by the kernel. As
+such, there are only two requirements: zswap enabled in the kernel and a
+swapfile or swap partition on the system.
+
+Several compression options are available for zswap. The choice is use-case
+dependent, with the primary trade-offs being speed or size. See [2] for example
+benchmarks of compressors in general.
+
+Ensure zswap (CONFIG_ZSWAP) support and a compressor are enabled in the kernel:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   Memory Management options --->                                             |
+|       <*> Compressed cache for swap pages                                    |
+|           Compressed cache for swap pages default compressor (xxx)           |
+|           Compressed cache for swap pages default allocator (xxx)            |
+|       <*> Enable the compression cache for swap pages by default             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Either set CONFIG_ZSWAP_DEFAULT_ON=y in the kernel config or add zswap_enabled=1
+to your kernel command line (via the bootloader or the built-in command line) to
+have zswap enabled at boot-time. To enable zswap at runtime,
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ echo 1 > /sys/module/zswap/parameters/enabled                            |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Zswap has many options that can also be configured at runtime, including which
```

Comment:
The zram pages don't capitalize `zram` but the zswap page does capitalize `zswap`. `Zswap` -> `zswap` to match the zram page?